cmd/invoke.go

```diff
@@ -83,7 +83,7 @@ dapr invoke --unix-domain-socket --app-id target --method sample --verb GET
 		if err != nil {
 			err = fmt.Errorf("error invoking app %s: %s", invokeAppID, err)
 			print.FailureStatusEvent(os.Stderr, err.Error())
-			return
+			os.Exit(1)
```

Comment:
@shubham1172 Can you add an e2e for this scenario?

Reply:
@mukundansundar added the test.